<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recordWindow</class>
 <widget class="QWidget" name="recordWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>540</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>历史记录</string>
  </property>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>55</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>500</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>660</width>
     <height>400</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchlineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>55</y>
     <width>240</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>支持游戏名模糊查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;游戏历史记录&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>700</width>
     <height>5</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
     <pointsize>9</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>返回游戏库</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>520</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>help-20.png</normaloff>help-20.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>searchlineEdit</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>returnButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
